### ISSUE TITLE
Photos disappear and popup appears while adding ingredients picture

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#476]
Photos (Front and nutrition pictures, added using camera) disappear and popup explaining the offline mode appears while adding ingredients picture.

Here's what I have in terms of logs.

```
01-17 13:09:57.209 848-7893/? I/ActivityManager: START u0 {cmp=openfoodfacts.github.scrachx.openfood/.views.SaveProductOfflineActivity (has extras)} from uid 10475 on display 0
01-17 13:09:57.475 14655-14655/openfoodfacts.github.scrachx.openfood D/SQL Log: SQLiteQuery: SELECT * FROM SEND_PRODUCT WHERE barcode = ?
01-17 13:09:57.628 14655-14655/openfoodfacts.github.scrachx.openfood W/PathParser: Points are too far apart 4.000000596046461
01-17 13:10:05.740 848-1382/? I/ActivityManager: START u0 {act=android.media.action.IMAGE_CAPTURE flg=0x3 cmp=com.sonyericsson.android.camera/.CameraActivityForCaptureOnlyPhoto clip={text/uri-list U:content://openfoodfacts.github.scrachx.openfood.easyphotopicker.fileprovider/picturesext/3625c5f5-d111-43d4-9e66-99dc3aaa1d16752418604.jpg} (has extras)} from uid 10475 on display 0
01-17 13:10:11.387 14655-17869/openfoodfacts.github.scrachx.openfood W/ExifInterface: Skip the tag entry since tag number is not defined: 2
01-17 13:10:15.733 848-1382/? I/ActivityManager: START u0 {act=android.media.action.IMAGE_CAPTURE flg=0x3 cmp=com.sonyericsson.android.camera/.CameraActivityForCaptureOnlyPhoto clip={text/uri-list U:content://openfoodfacts.github.scrachx.openfood.easyphotopicker.fileprovider/picturesext/a1204530-b97e-4ad9-8119-08c2996f0f16-823289542.jpg} (has extras)} from uid 10475 on display 0
01-17 13:10:19.768 14655-17206/openfoodfacts.github.scrachx.openfood W/ExifInterface: Skip the tag entry since tag number is not defined: 2
01-17 13:10:22.065 848-2665/? I/ActivityManager: START u0 {act=android.media.action.IMAGE_CAPTURE flg=0x3 cmp=com.sonyericsson.android.camera/.CameraActivityForCaptureOnlyPhoto clip={text/uri-list U:content://openfoodfacts.github.scrachx.openfood.easyphotopicker.fileprovider/picturesext/e5242c9a-3d54-4b0b-8100-3a74d5109390-1739393564.jpg} (has extras)} from uid 10475 on display 0
01-17 13:10:27.245 14655-14655/openfoodfacts.github.scrachx.openfood D/SQL Log: SQLiteQuery: SELECT * FROM SEND_PRODUCT WHERE barcode = ?
01-17 13:10:27.480 14655-14655/openfoodfacts.github.scrachx.openfood E/WindowManager: android.view.WindowLeaked: Activity openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity has leaked window com.android.internal.policy.PhoneWindow$DecorView{92e671a V.E...... R.....ID 0,0-720,467} that was originally added here
                                                                                          at android.view.ViewRootImpl.<init>(ViewRootImpl.java:368)
                                                                                          at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:299)
                                                                                          at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:85)
                                                                                          at android.app.Dialog.show(Dialog.java:319)
                                                                                          at com.afollestad.materialdialogs.MaterialDialog.show(MaterialDialog.java:1466)
                                                                                          at com.afollestad.materialdialogs.MaterialDialog$Builder.show(MaterialDialog.java:1457)
                                                                                          at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity.onCreate(SaveProductOfflineActivity.java:146)
                                                                                          at android.app.Activity.performCreate(Activity.java:6251)
                                                                                          at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
                                                                                          at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2397)
                                                                                          at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2504)
                                                                                          at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4105)
                                                                                          at android.app.ActivityThread.-wrap15(ActivityThread.java)
                                                                                          at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1373)
                                                                                          at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                          at android.os.Looper.loop(Looper.java:148)
                                                                                          at android.app.ActivityThread.main(ActivityThread.java:5461)
                                                                                          at java.lang.reflect.Method.invoke(Native Method)
                                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
01-17 13:10:27.517 14655-14655/openfoodfacts.github.scrachx.openfood D/SQL Log: SQLiteQuery: SELECT * FROM SEND_PRODUCT WHERE barcode = ?
01-17 13:10:27.646 14655-17207/openfoodfacts.github.scrachx.openfood W/ExifInterface: Skip the tag entry since tag number is not defined: 2
01-17 13:10:27.655 14655-14655/openfoodfacts.github.scrachx.openfood W/PathParser: Points are too far apart 4.000000596046461
01-17 13:10:27.714 848-3369/? I/WindowManager: Screen frozen for +665ms due to Window{1f2d94f u0 openfoodfacts.github.scrachx.openfood/openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity}
```

**Comment by teolemon** [comment]
This might be related to the order issue spotted in the PR @herau

**Comment by itchix** [comment]
Can I close?

**Comment by teolemon** [comment]
The popup has disappeared, but I still experience the issue.

**Comment by huzaifaiftikhar** [comment]
Closed in #1712
